CHANGELOG.md

```diff
@@ -8,7 +8,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 ## [unreleased] - YYYY-MM-DD
 
 ### Added
-
+- Added exporting model to ONNX format.
```

Comment:
Add the PR link here :]

Reply:
You can use suggestions here =)

Reply:
Didn't think of that. Never did that before 😅.

Reply:
Oops! missed that out. Will add :]